pkg/kustomize/kustomize_cli_test.go

```diff
@@ -0,0 +1,37 @@
+package kustomize_test
+
+import (
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"github.com/jenkins-x/jx/pkg/kustomize"
+)
+
+func TestKustomizeCLI_FindKustomize(t *testing.T) {
+	cwd, _ := os.Getwd()
+	tmpDir := filepath.Dir(filepath.Join(cwd, "..", "test_data", "kustomize_dummy", "base"))
+
+	k := kustomize.NewKustomizeCLI()
+	wantedOutput := []string{filepath.Join(tmpDir, "base/charts/kustomization.yaml"), filepath.Join(tmpDir, "base/kustomization.yaml"), filepath.Join(tmpDir, "staging/kustomization.yaml")}
+	output := k.FindKustomizationYamlPaths(tmpDir)
+
+	if !reflect.DeepEqual(wantedOutput, output) {
```

Comment:
Deep equal is not quite right here. You basically have to arrays and want to assert that both arrays have the same elements (not necessarily in order). Basically a set equal. There is an `assert.ElementsMatch` for that.

Reply:
```
	testDir := filepath.Join("test_data", "kustomize_dummy")
	absPath, err := filepath.Abs(testDir)
	require.NoError(t, err)
	wantedOutput := []string{
		filepath.Join(absPath, "base", "charts", "kustomization.yaml"),
		filepath.Join(absPath, "base", "kustomization.yaml"),
		filepath.Join(absPath, "staging", "kustomization.yaml"),
	}


	k := kustomize.NewKustomizeCLI()
	actualOutput := k.FindKustomizationYamlPaths(testDir)

	assert.ElementsMatch(t, wantedOutput, actualOutput)
```

Reply:
Provided you move the test data into the right directory.